db.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const mongoose = require('mongoose');
+
+const uri = 'mongodb://localhost/happy-mate';
+
+mongoose.connect(uri, {useMongoClient: true});
+
+module.exports = uri;
```

Comment:
This module shouldn't export a string since it doesn't really make sense when requiring this module. Instead, return a useful object.
```
module.exports = {
  uri,
};
```

```
// so in some other module (like test/tests/helper.js)
const clearDB  = require('mocha-mongoose')(db.uri);

Reply:
good point.  Done.